src/services/storage/storage-migration.ts

```diff
@@ -0,0 +1,3 @@
+export default function migrate(): Promise<void> {
```

Comment:
can you elaborate this function please?

Reply:
This function is intended to be used in future when there will be different versions of the extension in production. Whenever a new version is installed, this function should be called to migrate existing data in the extension storage and adjust it to the current version. For now this is not needed at all, but I wanted to create it for later.

Reply:
next time throw `Not Implemented` error in these kind of empty functions please. That also explains the situation for the code-reader.